composer.json

```diff
@@ -29,6 +29,7 @@
         "symfony-cmf/symfony-cmf": "^2.0",
         "symfony-cmf/seo-bundle": "2.0.*",
         "symfony-cmf/routing-auto-bundle": "2.0.*",
+        "symfony-cmf/sonata-phpcr-admin-integration-bundle": "^1.0",
```

Comment:
the `sonata-project/doctrine-phpcr-admin-bundle` dependency can now be removed (alongside with the `sonata-project/admin-bundle` dep)